source/NRF52TouchSensor.cpp

```diff
@@ -32,8 +32,9 @@ static NRF52TouchSensor *instance = NULL;
 
 static void touch_sense_irq(uint16_t mask)
 {
+    Event evt;
```

Comment:
Thanks for the PR @kevinawalsh!
Just a small thing, can we move this inside the if statement?

Reply:
Sure thing.